darwin/exporter/formats/yolo_segmented.py

```diff
@@ -180,13 +180,22 @@ def _handle_polygon(
 
     last_point = None
     try:
-        if "path" in data:
+        if "paths" in data:
+            paths_data = data["paths"]
+            if len(paths_data) > 1:
+                raise DarwinException from ValueError(
+                    "Complex polygon detected. The YOLOV8 format only supports simple polygons with a single path."
+                )
+            path_data = paths_data[0]
+        elif "path" in data:
+            # Continuing to support old version, in case anything relies on it.
```

Comment:
This fixes a bug with polygons in this conversion format. Since all Darwin JSON 2.0 polygons are represented as `paths` (even simple polygons with a single path), we need to make sure we don't ignore them